test/scenarios.py

```diff
@@ -5782,22 +5585,9 @@
                     # make sure we have received something, so that we have non-empty tmp file
                     action.Wait(event.Progress(0, FILES["testfile-big"].id)),
                     action.Stop(),
-                    action.Wait(event.FinishFailedTransfer(0, Error.CANCELED)),
                     action.WaitForAnotherPeer(),
                     # start the sender again
                     action.Start("172.20.0.5", dbpath="/tmp/db/29-1-ren.sqlite"),
-                    action.Wait(
-                        event.Queued(
-                            0,
-                            {
-                                event.File(
-                                    FILES["testfile-big"].id,
-                                    "testfile-big",
-                                    10485760,
-                                ),
-                            },
-                        )
-                    ),
                     action.Wait(event.Start(0, FILES["testfile-big"].id)),
```

Comment:
I'm not sure if this is correct. We expect `Start` to be emitted only once I guess.
We don't have `Paused` event to matchup with `Start`.

Reply:
Then i guess this is a bug in the implementation if we're shooting out start events every time

Reply:
Yeah, it might be